index.html

```diff
@@ -42,18 +42,18 @@ <h2>Select a category:</h2>
            <span> <img class="warning-icon visibility-hidden" src="assets/warning.svg" alt="error icon"></span>A category is required.</label>
            <ul class="activity-container">
             <li>
-              <label class="activity-input-label">What would you like to accomplish during this time?</label>
+              <label class="activity-input-label activity-user-input-label">What would you like to accomplish during this time?</label>
```

Comment:
Could this be activity text input label or description input label instead activity user input label